runtime/src/accounts_db.rs

```diff
@@ -61,6 +64,7 @@ const MAX_RECYCLE_STORES: usize = 1000;
 const STORE_META_OVERHEAD: usize = 256;
 const MAX_CACHE_SLOTS: usize = 200;
 const FLUSH_CACHE_RANDOM_THRESHOLD: usize = MAX_LOCKOUT_HISTORY;
+const SCAN_SLOT_PAR_ITER_THRESHOLD: usize = 4000;
```

Comment:
Based on benches introduced in `setup_bench_dashmap_iter`, the number is higher for machines with greater number of cores (more Rayon overhead), but this is a good average of results on machines with 16-48 core performance.